blog/2019/06/09/site-update.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Post-migration updates & clean-up
+tags: news
+---
+
+Over the course of the past weeks many clean-ups have been
+done, reaping the benefits provided by our new
+[Statocles](https://preaction.me/statocles)-based site:
+
+---
+
+* Added back the ability maintain News on the site
+* Remove "Fork me on GitHub" banner for improved mobile usability
+* Cleaned up GitHub repository ('master' branch) from all unused parts
```

Comment:
I'm not sure if the removal is complete in the sense, in the sense of

> all unused parts

Reply:
Rephrased to remove the inclination to add the "all" trigger-word.